internal/containerd/install.go

```diff
@@ -127,3 +132,75 @@ func isContainerdNotInstalled() bool {
 	_, runcNotFoundErr := exec.LookPath(runcPackageName)
 	return containerdNotFoundErr != nil || runcNotFoundErr != nil
 }
+
+// Client is a containerd runtime client wrapper
+// Holds the internalapi.RuntimeService for pod/container operations
+type Client struct {
+	Runtime internalapi.RuntimeService
+}
+
+// NewClient creates a new Client with a real containerd runtime service
+func NewClient() (*Client, error) {
+	runtime, err := remote.NewRuntimeService(ContainerRuntimeEndpoint, 5*time.Second)
+	if err != nil {
+		return nil, err
+	}
+	return &Client{Runtime: runtime}, nil
+}
+
+func (c *Client) RemovePods() error {
+	podSandboxes, err := c.Runtime.ListPodSandbox(&v1.PodSandboxFilter{
```

Comment:
for these containerd calls, do we need to add the namespace somewhere, or does it default to the k8s.io one?

Reply:
This actually uses the cri-api, which only works with the k8s.io namespace.